src/installer.ts

```diff
@@ -21,15 +21,52 @@ function voltaVersionHasSetup(version: string): boolean {
   return semver.gte(version, '0.7.0');
 }
 
-export function buildDownloadUrl(platform: string, version: string): string {
+async function getOpenSSLVersion(version = ''): Promise<string> {
```

Comment:
Can we split this into two functions, one that calls `openssl version` and another that parses the result?  That way we can make some nice unit tests for various values that `openssl version` returns (e.g. test that regexp on line 43)...

What do you think?

Reply:
I was going to do that, but figured it was all related to getting the version, so kept them together. I can totally separate them.

Reply:
I split them up! At the least it's more legible, though there wasn't much value in adding more tests.